Doppler.HtmlEditorApi/Controllers/CampaignsController.cs

```diff
@@ -133,9 +133,13 @@ public async Task<IActionResult> SaveCampaign(string accountName, int campaignId
             else
             {
                 await _campaignContentRepository.CreateCampaignContent(accountName, contentRow);
+                // We are using HTMLSourceType = 2 (Template) because Editor seems to be tied to the old HTML Editor
+                // See https://github.com/MakingSense/Doppler/blob/48cf637bb1f8b4d81837fff904d8736fe889ff1c/Doppler.Transversal/Classes/CampaignHTMLContentTypeEnum.cs#L12-L17
+                await _campaignContentRepository.UpdateCampaignStatus(2, 2, contentRow.CampaignId, 1);
             }
             await _campaignContentRepository.SaveNewFieldIds(campaignId, fieldIds);
             await _campaignContentRepository.SaveLinks(campaignId, trackableUrls);
+            await _campaignContentRepository.UpdateCampaignPreviewImage(campaignId, contentRow.PreviewImage);
```

Comment:
I think that it is not important, but executing the `UpdateCampaignPreviewImage` before would be more similar to the previous behavior, in my opinion, it is less risky.

```suggestion
            await _campaignContentRepository.UpdateCampaignPreviewImage(campaignId, contentRow.PreviewImage);
            await _campaignContentRepository.SaveNewFieldIds(campaignId, fieldIds);
            await _campaignContentRepository.SaveLinks(campaignId, trackableUrls);
```

Reply:
Fixed